src/components/ChatProvider/index.tsx

```diff
@@ -1,15 +1,87 @@
-import React, { createContext, useState } from 'react';
+import React, { createContext, useCallback, useEffect, useRef, useState } from 'react';
+import { Client } from '@twilio/conversations';
+import { Conversation } from '@twilio/conversations/lib/conversation';
+import { Message } from '@twilio/conversations/lib/message';
```

Comment:
The conversations library doesn't have these exported by default? It feels weird accessing them directly from their internal folders.

Reply:
I had the same thought, but it looks like it's not possible to import these types from `@twilio/conversations` directly.